sdk/python/feast/feature_server.py

```diff
@@ -100,8 +100,10 @@ def async_refresh():
     @asynccontextmanager
     async def lifespan(app: FastAPI):
         async_refresh()
+        await store.initialize()
         yield
         stop_refresh()
+        await store.close()
```

Comment:
can create and cleanup resources as part of the feature server lifespan

Reply:
Can you remove this and let's rerun the integration test? Or maybe create a separate branch without it?